wiki/Tournaments/CJB/2018/4K/en.md

```diff
@@ -0,0 +1,145 @@
+     - All maps are in a FreeMod bracket.
+3. Match schedule will be settled by the Tournament Management.
+4. If no staff or referee is available, the match will be postponed.
+5. Failed during a map will be treated as losing in this map, even though you have a higher score.
+6. Use of the [Visual Settings](/wiki/Game_Modifiers) are allowed.
+7. If the beatmap ends in a draw, the game will be nullified.
+8. If player disconnects (In the case of the player is able to finish the map), the disconnected player will have to upload the score (`Shift` + `F12`) and provide the link to match referee instantly.
+9. If a player disconnects (In the case of the player is not able to finish the map):  
+     - disconnets within 30s after the match starts, the match will be reset.   
+     - the player will be recorded as Failed.
+10. Beatmaps cannot be reused in the same match unless the beatmap was nullified.
+11. If not all players attend, the maximum time the match can be postponed is 10 minutes. Absent player(s) will be considered as forfeited.
+12. All players are supposed to keep the match running fluent and without delays. The match will not be reset if the players have (including but not limited to):
+     - Internet issue.
+     - System notification issue.
+     - Hardware problems.
+13. Unexpected incidences are handled by the tournament management.
+14. Any modification of these rules will be announced.
+15. The CJB team reserves the right to use Chinese preferentially.
+
+### Registration Rules
+
+1. Everyone who interested in this tournament should join the [Tournament QQ Group](https://jq.qq.com/?_wv=1027&k=5sTdKLp) and read the announcement in the group to register.
+2. In order for your registration to count, you must meet following conditions:
+    - Participant's flag on their profile must be a country from **Asia**.
+    - Participant uses Chinese as the primary language.
+    - Participant **should not get more than 285pp on any 4K map** _and_ **the sum of your five best 4K maps' pp is not greater than 1350**.
+    - If player has record of violating the osu! community rules in the last 12 months, we may cancel the application of this player.
+    - The quota of this tournament is **16**, first come first served.
+3. Successfully registered player will receive a confirm email and will be posted on the players list.
+4. Map Selectors and Managers are not allowed to register.
+
+### Mappool Instructions
+
+1. Each stage has **1** pool:
+    - Pool of the Round of 16 and the Quarterfinals has **13** maps.
+    - Pool of the Semifinals and the Finals has **15** maps.
+2. Each mappool has one TieBreaker.
+3. All of the maps will be played under _Free-Mod_ conditions.
+
+### Match Instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period:
+     - Room settings are osu!mania, Head To Head, Win Condition: 'Score v2'. Room name must be "CMT4K2017: (playerBlue) vs (playerRed)".
```

Comment:
`Room name must be` -> `Room name is`

I think the room name itself is copied and pasted. That's the wrong tournament.